charts/kubewarden-crds/templates/policyservers.yaml

```diff
@@ -1437,6 +1437,32 @@ spec:
                           Note that this field cannot be set when spec.os.name is windows.
                         format: int64
                         type: integer
+                      seLinuxChangePolicy:
```

Comment:
why did we get that?

Reply:
I think it comes from bumping the `render.kubernetesVersion` from v1.30 to v1.31 in https://github.com/kubewarden/kubewarden-controller/pull/947/files#diff-e34d8d9c8da7fb4ce84e8c50319f3e1d3f0ddc42e569d86cff67153b84a1f0e7R11

Reply:
I see, fine